mm2src/coins/utxo/utxo_builder/utxo_coin_builder.rs

```diff
@@ -818,27 +818,24 @@ async fn check_electrum_server_version(
 
 /// Wait until the protocol version of at least one client's Electrum is checked.
 async fn wait_for_protocol_version_checked(client: &ElectrumClientImpl) -> Result<(), String> {
-    let mut attempts = 0;
-    loop {
-        if attempts >= 10 {
-            return ERR!("Failed protocol version verifying of at least 1 of Electrums in 5 seconds.");
-        }
-
+    repeatable!(async {
         if client.count_connections().await == 0 {
             // All of the connections were removed because of server.version checking
-            return ERR!(
+            ready!(ERR!(
                 "There are no Electrums with the required protocol version {:?}",
                 client.protocol_version()
-            );
+            ));
         }
 
         if client.is_protocol_version_checked().await {
-            break;
+            ready!(Ok(()));
```

Comment:
I think break should have a name similar to break...i.e `break!()`. just to be clear instead of using `ready!()`

Reply:
Do you mean renaming `ready!()` to `break!()`?
In my opinion, `break` doesn't make sense in terms of a repeatable future, because there is no a loop:
```rust
let _res = repeatable!(async {
  if client.is_protocol_version_checked().await {
    break!(Ok(()));
  }
})
.attempts(10)
.await;
```
What do you think?

Reply:
agreed! let's go with `ready!()` then..